dd-bridge-android/src/main/kotlin/com/datadog/android/bridge/internal/BridgeSdk.kt

```diff
@@ -106,6 +106,12 @@ internal class BridgeSdk(
         } else {
             configBuilder.useViewTrackingStrategy(NoOpViewTrackingStrategy)
         }
+
+        val longTask = configuration.additionalConfig?.get(DD_LONG_TASK_THRESHOLD) as? Long
```

Comment:
that is an interesting topic by the way, because there is native main thread - where we track long tasks by default and which is essentially responsible mainly for UI.

But since it is React Native, there is also JS VM with its own thread loop, where execution of JS code happens, which I guess it not the same as main thread in Android JVM, since RN has dedicated VM for its own stuff. We may be interested in tracking it somehow. And afaik RN is using this https://yogalayout.com/ for making layout async (so it off-loads some work from native main thread).

Reply:
Yes, tracking JS side long tasks is another topic we'll work on eventually